dm/syncer/err-operator/operator.go

```diff
@@ -34,11 +34,10 @@ import (
 // Operator contains an operation for specified binlog pos
 // used by `handle-error`.
 type Operator struct {
-	uuid          string // add a UUID, make it more friendly to be traced in log
-	op            pb.ErrorOp
-	isAllInjected bool                       // is all injected, used by inject
-	events        []*replication.BinlogEvent // startLocation -> events
-	originReq     *pb.HandleWorkerErrorRequest
+	uuid      string // add a UUID, make it more friendly to be traced in log
+	op        pb.ErrorOp
+	events    []*replication.BinlogEvent // startLocation -> events
```

Comment:
```suggestion
	events    []*replication.BinlogEvent //  events sorted by startLocation
```

Reply:
`events` is constructed by ddls in `binlog inject ddl1, ddl2....`. it will add `LogPos ` and `EventSize ` when `MatchAndApply` and the `LogPos ` will be differnet because of `replace` or `inject`.

Reply:
ohh i see, this commnet `startLocation -> events` is misunderstood 😷

Reply:
Maybe `ddls -> events` is better, because most of events is  constructed by ddls.